src/GuestConfiguration/GuestConfiguration/Common/GuestConfigurationCmdletBase.cs

```diff
@@ -271,13 +278,18 @@ private string GetPolicySetCategory(PolicySetDefinition initiativeDefinition)
         {
             var categoryMetadata = JObject.Parse(initiativeDefinition.Metadata.ToString());
             var categoryMetadataDictionary = categoryMetadata.ToObject<Dictionary<string, object>>();
-            var category = categoryMetadataDictionary["category"].ToString();
-            return category;
+            if (categoryMetadataDictionary.ContainsKey("category"))
+            {
+                var category = categoryMetadataDictionary["category"].ToString();
+                return category;
+            }
+
+            return string.Empty;
         }
 
         private IEnumerable<PolicySetDefinition> GetAllGuestConfigPolicySetDefinitions()
         {
-            var policySetDefinitions = PolicyClient.PolicySetDefinitions.ListBuiltIn();
+            var policySetDefinitions = PolicyClient.PolicySetDefinitions.List();
```

Comment:
List returns custom policies and excludes builtin policies. can you make sure after cmdlet works on builtin initiatives as well with your changes ?